korman/operators/op_mesh.py

```diff
@@ -0,0 +1,369 @@
+                                             description="Generates helper objects where rungs should be placed",
+                                             default=True)
+    rung_width_type = bpy.props.EnumProperty(name="Rung Width",
+                                             description="Type of rungs to generate",
+                                             items=[("FULL", "Full Width Rungs", "The rungs cross the entire width of the ladder"),
+                                                    ("HALF", "Half Width Rungs", "The rungs only cross half the ladder's width, on the side where the avatar will contact them"),],
+                                             default="FULL")
+    # Game options
+    has_upper_entry = bpy.props.BoolProperty(name="Has Upper Entry Point",
+                                             description="Specifies whether the ladder has an upper entry",
+                                             default=True)
+    upper_entry_enabled = bpy.props.BoolProperty(name="Upper Entry Enabled",
+                                                 description="Specifies whether the ladder's upper entry is enabled by default at Age start",
+                                                 default=True)
+    has_lower_entry = bpy.props.BoolProperty(name="Has Lower Entry Point",
+                                             description="Specifies whether the ladder has a lower entry",
+                                             default=True)
+    lower_entry_enabled = bpy.props.BoolProperty(name="Lower Entry Enabled",
+                                                 description="Specifies whether the ladder's lower entry is enabled by default at Age start",
+                                                 default=True)
+
+    def draw(self, context):
+        layout = self.layout
+        space = bpy.context.space_data
+        if (not space.local_view):
+            box = layout.box()
+            box.label("Geometry:")
+
+            row = box.row()
+            if (self.ladder_height % 2 != 0):
+                row.alert = True
+            row.prop(self, "ladder_height")
+            row = box.row()
+            row.prop(self, "ladder_width")
+            row = box.row()
+            row.prop(self, "rung_height")
+
+            box = layout.box()
+            box.label("Template Guides:")
+            col = box.column()
+            col.prop(self, "gen_back_guide")
+            col.prop(self, "gen_ground_guides")
+            col.prop(self, "gen_rung_guides")
+            if self.gen_rung_guides:
+                col.separator()
+                col.prop(self, "rung_width_type", text="")
+
+            box = layout.box()
+            row = box.row()
+            col = row.column()
+            col.label("Upper Entry:")
+            col.row().prop(self, "has_upper_entry", text="Create")
+            row = col.row()
+            if not self.has_upper_entry:
+                row.enabled = False
```

Comment:
Could be simplified to `row.enabled = not self.has_upper_entry`

Reply:
Your logic is reversed, but good idea! :trollface: